cmd/brimstone/main.go

```diff
@@ -126,23 +118,30 @@ func main() {
 			return len(ggsig) > 0
 		},
 		Validator: func(key string, c echo.Context) (bool, error) {
-			brimstoneapikey, ok := GetAPIKey()
-			return (ok && key == brimstoneapikey), nil
+			return key == cfg.ApiKey, nil
```

Comment:
since the echo context is passed in, and the config was added to the context; recommend to get api key from the context-config, smth like c.Get("config").APIKey